website/src/utils/networkUtils.ts

```diff
@@ -0,0 +1,139 @@
+// Networks that should use the "mono" icon variant
+export const MONO_ICON_NETWORKS = [
+  'vana',
+  'vana-moksha',
+  'xlayer-mainnet',
+  'xlayer-sepolia',
+  'zksync-era',
+  'zksync-era-sepolia',
+  'sonic',
+  'soneium-testnet',
+  'soneium',
+  'linea-sepolia',
+  'linea',
+  'lens-testnet',
+  'expchain-testnet',
+  'autonomys-taurus',
+  'arweave-mainnet',
+  'fraxtal',
+  'lumia',
+  'mbase',
+]
+
+// Skeleton networks (no icon available)
+export const MISSING_ICON_NETWORKS = [
+  'berachain-bepolia',
+  'hoodi',
+  'ink-sepolia',
+  'megaeth-testnet',
+  'solana-accounts',
+  'manta',
+]
+
+// Non-EVM networks
+export const NON_EVM_NETWORKS = [
```

Comment:
Like AK said, I am pretty sure we have this information in the registry already, or can infer it based on something more general than specific network names/IDs?